### PR TITLE
Update Rule “practice-makes-perfect/rule”

### PR DESCRIPTION
**Pull request by leehawkinsssw**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by UlyssesMaclaren — CHANGES_REQUESTED** [review]
I'd change from 
    Moving of your hands less/more?
to
    Moving your hands less/more?

**Comment by leehawkinsssw** [comment]
yep, mea culpa when changing what was originally there

File: rules/practice-makes-perfect/rule.md [hunk 1]
```diff
@@ -11,45 +10,46 @@ authors:
...
 
    ::: info
-   Tip: PowerPoint’s "Rehearse with coach" will give you immediate feedback when practicing.
+   Tip: PowerPoint’s "Rehearse with Coach" will give you immediate feedback when practising.
    :::
 
 3. Do a visual audit – play the recording, this time looking at the video only (turn down the volume to zero, so that you can’t hear the audio).
 
-   👀 Check: How is your body language?, movement of your hands less/more? (more hand movement means more distraction) Eye contact? Posture? Are you smiling or do you look stunned?
+   👀 Check: How is your body language? Moving your hands less/more? (more hand movement means more distraction) Eye contact? Posture? Are you smiling or do you look stunned?
```
Comment (wicksipedia) [inline rules/practice-makes-perfect/rule.md:41]:
```suggestion
   👀 Check: How is your body language? Do stuff (more hand movement means more distraction) Eye contact? Posture? Are you smiling or do you look stunned?
```

**Review by UlyssesMaclaren — COMMENTED** [review]
Nice :)

**Pull request closed without merging**